src/SavedSearch.php

```diff
@@ -463,19 +463,7 @@ public function cleanDBonPurge()
     }
 
 
-    /**
-     * Print the saved search form
-     *
-     * @param integer $ID      ID of the item
-     * @param array   $options possible options:
-     *                         - target for the Form
-     *                         - type when adding
-     *                         - url when adding
-     *                         - itemtype when adding
-     *
-     * @return void
-     **/
-    public function showForm($ID, array $options = [])
+    public function showForm($ID, array $options = []): bool
```

Comment:
I am not in favor of changing the signature in the 10.0/bugfixes branch if it is not required.

```suggestion
    public function showForm($ID, array $options = [])
```